### ISSUE TITLE
/dcc send gives "Received malformed DCC request ..." on destination machine

### ISSUE THREAD

**Issue by anotheranne** [weechat/weechat#782]
weechat 1.6-dev compiled Aug 21 2016 11:28:12

unable to send files to another machine on network. (able to do so with other XChat).
at command
/dcc send noza xfertest
the destination machine shows error message on channel.

noza |  Received a malformed DCC request from inksi.                                                         
noza | * Contents of packet: DCC SEND xfertest 3232235542 6660 0    

Xchat pop-up box showing details of transfer does not pop up
xfer.list buffer on Weechat shows 'connecting' which eventually times out to show 'failed'

Ian

**Comment by anotheranne** [comment]
also noted that files sent to weechat are 'failed' instantly

Ian

**Comment by talisein** [comment]
Looks like xchat will reject 0 sized files. Does a non-zero sized file send ok?

**Comment by anotheranne** [comment]
Yes, sends fine, but receives are still an issue, I will check through to see if some setting is affecting this. Just made a test file with vim. should have put a couple of characters in.

sorry about that.

Ian

**Comment by talisein** [comment]
So you have a problem sending from XChat and receiving on weechat that is different from #781?

**Comment by anotheranne** [comment]
yes. this is closed as it must be an Xchat issue.
i am investigating the sending from Xchat to Weechat.
